src/components/action/style.css

```diff
@@ -1,8 +1,98 @@
 @import "../../assets/common.css";
 
-:host {
-	display: block;
-}
 :host[hidden] {
 	display: none;
 }
+button {
+	border: 1px solid transparent;
+	border-radius: 3px;
+	padding: .2em .3em;
+	font-size: 110%;
+	font-weight: 400;
+}
+:host[processing] > button {
+	color: var(--smoothly-light-contrast);
+	background: var(--smoothly-light-color);
+	border-color: var(--smoothly-light-color);
+}
+:host:not([color])
+:host[color="primary"] {
+	--smoothly-submit-color: var(--smoothly-primary-color);
```

Comment:
Variable should be contain `action` instead of `submit`.

Reply:
Naturally